src/cdk/jsii-project.ts

```diff
@@ -425,6 +425,11 @@ export class JsiiProject extends TypeScriptProject {
 
       // https://github.com/projen/projen/issues/2264
       this.package.addPackageResolutions("@types/babel__traverse@7.18.2");
+
+      this.addDevDeps(
+        // https://github.com/projen/projen/pull/3076
+        `@types/node@^${major(this.package.minNodeVersion ?? "16.0.0")}`
+      );
```

Comment:
Can you make this conditional? Only do this if we are on 1.x
```suggestion
      if ((options.jsiiVersion ?? "1.x").startsWith('1.x')) {
        this.addDevDeps(
          // https://github.com/projen/projen/pull/3076
          `@types/node@^${major(this.package.minNodeVersion ?? "16.0.0")}`
        );
      }
```

Reply:
Fixed.